taker-electron/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
why not trim trailing whitespaces in md files? trailing whitespaces are universally bad 😝

Reply:
outdated file from the boilerplate. Thanks. I'll delete it.